src/js/ui/user/variantperf/variantPerfView.js

```diff
@@ -169,7 +173,104 @@ function isEmpty(element) {
     return '';
 }
 
-function toTitleCase(str)
-{
+function toTitleCase(str) {
     return str.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
 }
+
+function createGraph(data, node) {
+  const ctx = node.dom.getContext('2d');
+  drawChart(data, ctx);
+  node.state.hash = chartHash(data);
+}
+
+function updateGraph(data, node) {
```

Comment:
I think the whole `updateGraph` logic is useless here, since the perf screen don't auto update. Each time you go again to the screen it will be updated, so only the `oninit` logic is needed.